### PR TITLE
Relocate ActionMailbox

### PR DESCRIPTION
**Pull request by mullermp**
Relocates the ActionMailbox SES feature to https://github.com/aws/aws-actionmailbox-ses-ruby/pull/1

### CODE REVIEW

**Comment by mullermp** [comment]
@ssunday fyi

File: Gemfile [hunk 1]
```diff
@@ -6,6 +6,7 @@ gemspec
 
 gem 'aws-actiondispatch-dynamodb', git: 'https://github.com/aws/aws-actiondispatch-dynamodb-ruby'
 gem 'aws-actionmailer-ses', git: 'https://github.com/aws/aws-actionmailer-ses-ruby'
+gem 'aws-actionmailer-ses', git: 'https://github.com/aws/aws-actionmailer-ses-ruby', branch: 'init'
```
Comment (ssunday) [inline Gemfile:9]: ugh, shouldn't this be MAILBOX?

**Review by ssunday — CHANGES_REQUESTED** [review]

File: Gemfile (hunk 1 above)
Comment (mullermp) [reply Gemfile]: Yeah. Copy paste error... fixed it

File: sample-app/config/initializers/action_mailer.rb [hunk 2]
```diff
@@ -1,3 +1,3 @@
 options = {}
-ActionMailer::Base.add_delivery_method :ses, Aws::ActionMailer::SESMailer, **options
-ActionMailer::Base.add_delivery_method :ses_v2, Aws::ActionMailer::SESV2Mailer, **options
+ActionMailer::Base.add_delivery_method :ses, Aws::ActionMailer::SES::Mailer, **options
```
Comment (ssunday) [inline sample-app/config/initializers/action_mailer.rb:2]: isn't this an unrelated to mailbox change

**Review by ssunday — COMMENTED** [review]
Sqlite can be killed from the gemfiles/X, no?

File: sample-app/config/initializers/action_mailer.rb (hunk 2 above)
Comment (mullermp) [reply sample-app/config/initializers/action_mailer.rb:2]: It's in the sample-app. A change I made for the other gem. I just forgot to change it here, and when testing ActionMailbox, I got an unrelated error.

**Comment by mullermp** [comment]
>Sqlite can be killed from the gemfiles/X, no?

Yes. I can do this now. My original plan was to just do a sweeping clean up after I got all of the functionality out.

**Review by alextwoods — APPROVED** [review]

**Comment by mullermp** [comment]
@ssunday All approved from our side. I think you can actually test this if you pull in aws-sdk-rails from github/branch in your gemfile, this gem and the other gems the same way. It should all still work. Otherwise, we can defer that and if you want to give approves that's fine too.

**Pull request merged**